REQUIRE

```diff
@@ -1,6 +1,6 @@
 julia 0.6
-AWSCore 0.3
-SymDict 0.1.2
+AWSCore 
+SymDict 
 Retry
 MbedTLS
-HTTP 0.5.4
+HTTP 
```

Comment:
Was the presence of the minimum versions in REQUIRE causing a problem for you?

Reply:
probably not, I did not know that this is minimum version. was thinking about required version.

Reply:
added back of the tags.